src/tests/mod.rs

```diff
@@ -15,17 +15,19 @@ mod literals;
 mod operators;
 
 fn eval_ok(nix_expr: &str) -> Value {
-    match evaluate(nix_expr) {
+    let workdir = std::env::current_dir().unwrap();
+    match evaluate(nix_expr, &workdir) {
         Ok(val) => val,
         Err(err) => panic!(
-            "eval '{}' shouldn't fail.\nError message: {}",
+            "eval '{}' shouldn't fail.\nError message: {:?}",
```

Comment:
Total nitpick (feel free to ignore or do in a separate PR):
```suggestion
            "eval '{nix_expr}' shouldn't fail.\nError message: {err:?}",
```
Then the line below can be removed.